src/Gateways/PorticoConnector.ts

```diff
@@ -1288,4 +1307,16 @@ export class PorticoConnector extends XmlGateway implements IPaymentGateway {
 
     return result;
   }
+
+  protected hydrateBatchHistorySummary(root: Element): BatchHistory {
+    const result = new BatchHistory();
+
+    result.date = root.findtext('.//CloseUtcDT');
+    result.approvalNumber = root.findtext('.//BatchSeqNbr');
```

Comment:
I tried to intuit if my mapping of the api fields to what the jira ticket wants are correct.

@lundhjem any insight if I got these corresponding to the right fields?

Reply:
https://cert.api2.heartlandportico.com/Gateway/PorticoSOAPSchema/build/Default/webframe.html#Portico_xsd~e-PosResponse~e-Ver1.0~e-Transaction~e-ReportBatchHistory.html

Reply:
@darrenjaworski I'd rather call them exactly what they are instead of us guessing at it. Asking product for clarification.